src/citations.jl

```diff
@@ -12,8 +12,7 @@ function get_citation(pkg)
         try
             import_bibtex(bib_path)
         catch e
-            @warn("There was an error reading the CITATION.bib file for $(pkg.name)")
-            @debug e
+            @warn("There was an error reading the CITATION.bib file for $(pkg.name)" exception=e)
```

Comment:
Arguably this should bne
```suggestion
            @error("There was an error reading the CITATION.bib file for $(pkg.name)" exception=e)
```

This was an error, we were not even able to recover from it correctly.
we kept running cos we know it is safe, but the final output is wrong (because it is incomplete)